src/redturtle/gallery/browser/templates/gallery_view.pt

```diff
@@ -16,7 +16,6 @@
 <metal:main_macro define-macro="content-core"
                   tal:define="images view/album_images">
 
-
```

Comment:
I wouldn't add a changed file to the commit if removing an empty line is the only change. Just to be as clean as possible in the changes.